exporter/lokiexporter/config.go

```diff
@@ -53,6 +53,10 @@ type Config struct {
 }
 
 func (c *Config) Validate() error {
+	if err := c.QueueSettings.Validate(); err != nil {
+		return fmt.Errorf("queue settings has invalid configuration: %w", err)
```

Comment:
Nitpick, but I think calling out the failing config key here would be helpful.

Reply:
the failing key could be QueueSize only, and it will be mentioned in the inner error message: https://github.com/open-telemetry/opentelemetry-collector/blob/7d168dd20efd4ebf659ccf7c501fdf02d7ed6dfd/exporter/exporterhelper/queued_retry.go#L76

So the error message here will be:
`queue settings has invalid configuration: queue size must be positive`

I think the message identifies the failing key clearly. Or maybe I misunderstood your concern?